code/include/rtweekend.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <cmath>
+#include <limits>
+#include <memory>
+#include <random>
+
+// Usings
+
+using std::shared_ptr;
+using std::make_shared;
+using std::sqrt;
+
+// Constants
+
+const double infinity = std::numeric_limits<double>::infinity();
+const double pi = 3.1415926535897932385;
+
+// Utility Functions
+
+inline double degrees_to_radians(double degrees) {
+    return degrees * pi / 180.0;
+}
+
+inline double random_double() {
```

Comment:
For the naming convention of inline functions, shouln't we treat them as normal functions and use CamerCase?

Reply:
You're right. Fixed it!